src/components/RangeSlider/components/DualThumb/DualThumb.scss

```diff
@@ -154,6 +142,16 @@ $range-accessibility-input-margin: rem(8px);
 $range-output-size: rem(32px);
 $range-output-tip-size: rem(8px);
 
+.Prefix {
+  flex: 0 0 auto;
+  margin-right: spacing(tight);
+}
+
+.Suffix {
+  flex: 0 0 auto;
+  margin-left: spacing(tight);
+}
+
```

Comment:
The `prefix` and `suffix` were previously passed to the text fields. It seemed like a good idea to add them back to the range slider, which means including some styles.